src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -88,42 +88,22 @@ def init_app(self, app):
             for (name, configuration) in self._provider_configurations.items()
         }
 
-    def _get_post_logout_redirect_uri(self, client):
-        if client.post_logout_redirect_uris:
```

Comment:
As this method is no longer required, add a test case to cover `post_logout_redirect_uris` property of `PyoidcFacade`. Take it from my PR.

```python
@property
def post_logout_redirect_uris(self):
    return self._client.registration_response.get('post_logout_redirect_uris')

def test_property_post_logout_redirect_uris(self):
    post_logout_redirect_uris = ['https://client.example.com/logout']
    client_metadata = self.CLIENT_METADATA.copy(
        post_logout_redirect_uris=post_logout_redirect_uris)
    facade = PyoidcFacade(ProviderConfiguration(provider_metadata=self.PROVIDER_METADATA,
        client_metadata=client_metadata),
                              REDIRECT_URI)
    assert facade.post_logout_redirect_uris == post_logout_redirect_uris
```

Reply:
Fixed! 👍